portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,40 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  //const greetings =
+  //    ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+  const imagesArray = 
+  ['images/youtube_rewind.png','images/chrome_ram.gif', 'images/coffee.png','images/sundar.png'];
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  var i;
+  const img = document.getElementById('greeting-container');
+  for (i=0; i<imagesArray.length; i++)
+  {
+      if(img.src.match(imagesArray[i]))
+      {
+          if (i == (imagesArray.length)-1)
+          {
+            img.src = imagesArray[0];
+          }
+          else
+          {
+          
+          img.src = imagesArray[i+1];
```

Comment:
to keep things super-duper clean you should delete the blank lines around this and indent this line properly.